Update find, favorite, goto, undo, redo sections for UG (#151)
AY1920S1-CS2103T-T13-4/main: docs/UserGuide.adoc

Clears all existing tags from the 2nd bookmark.

[[locating-bookmarks]]
=== Locating bookmarks by name, URL, tag or folder: *`find`*

Finds bookmarks that contain any of the given keywords in their name or
URL, or that are tagged with the given tag(s).

Format: `*find* [KEYWORD]... [t/TAG]... [f/FOLDER]...`

****
* At least one keyword, tag or folder must be provided.
* Keyword search is case insensitive. e.g video will match Video.
* The order of the keywords does not matter. e.g. `Website Video` will match `Video Website`.
* Keywords are only searched in the URL and name.
* For keywords, words will be partially matched e.g. `Video` will match `Videos`.
* For tags and folders, words will be fully matched.
* Bookmarks matching at least one search criterion will be returned (i.e. OR search).
****

Examples:

> * `*find* Website Video will t/Fun` +

Reviewer: actually what's the will for?

> * `*find* Website Video will t/Fun` +

Author: ```suggestion
* `*find* Website Video t/Fun` +
```

> Returns CS2103T Website, Video Streaming, and all bookmarks tagged with Fun.

Author: ```suggestion
Returns all the bookmarks with CS2103T or Video in its name or URL, and all bookmarks tagged with Fun.
```

Adds a Favorite tag to the specified bookmark.

Format: `*favorite|fav* INDEX`

****
* `fav` is an alias for `favorite`
* Removing the Favorite tag is the same as removing a normal tag from a bookmark (see <<editing-bookmarks>>)
* The index refers to the index number shown in the displayed bookmark list.
* The index must be a positive integer `1`, `2`, `3`, …​
****

Examples:

* `*favorite* 1` +
Adds a Favorite tag to the first bookmark in the bookmark manager.

* `*fav* 2` +
Adds a Favorite tag to the second bookmark in the bookmark manager.

=== Clearing all entries: *`clear`*

Clears all entries from the bookmark manager.

Format: `*clear*`

=== Undoing previous commands: *`undo`*

Restores the list to the state before the given number of previous undoable commands were
executed.

Format: `*undo* [STEP=1]`

//TODO:Check whether reminder supports undo/redo, include other
//undoable commands as well
****
* Undoable commands include commands that modify the bookmark list or
reminders, which includes `add`, `edit`, `delete`, `clear`, `reminder`,
`reminder edit`, `reminder delete`, etc).
* The step must be a positive integer `1`, `2`, `3`, …​
****

Examples:

* `*undo*` +

> Restores the list to the state before the previous undoable command was	executed.

Reviewer 2: ```suggestion
Restores the list to the state before the previous undoable command was executed.
```
